kahuna/public/js/image/controller.js

```diff
@@ -74,9 +74,24 @@ image.controller('ImageCtrl', [
         // initially)
         ctrl.cropKey = cropKey;
 
+        var addSpacing = function(string){
+        // If styling changes to capitalised first letters use:
+            // var firstLetter = string.charAt(0).toUpperCase();
+            // var restOfWord = string.slice(1).replace(/([A-Z]+)/g, $1 => ' ' + $1.toLowerCase());
+            // return firstLetter.concat(restOfWord);
+
+            return string.replace( /([A-Z]+)/g, $1 => ' ' + $1.toLowerCase() );
+        };
+
         // Alias for convenience in view
         ctrl.metadata = image.data.metadata;
 
+        ctrl.spacedMetadata = {};
+
+        for(var key in image.data.metadata) {
```

Comment:
And you might want to use `let key` for better block scoping if you do it this way.